### PR TITLE
Do not create dir if it already exists

### PR DESCRIPTION
**Pull request by spacecowboy**

### CODE REVIEW

File: start-mailserver.sh [hunk 1]
```diff
@@ -25,7 +25,12 @@ if [ -f /tmp/postfix/accounts.cf ]; then
     echo "${pass}" | userdbpw -md5 | userdb ${login} set systempw
     echo "${pass}" | saslpasswd2 -p -c -u ${domain} ${login}
     mkdir -p /var/mail/${domain}
-    maildirmake /var/mail/${domain}/${user}
+    if [ -d "/var/mail/${domain}/${user}" ]; then
```
Comment (tomav) [inline start-mailserver.sh]: Why not only:

    if ! [ -d "/var/mail/${domain}/${user}" ]; then
       maildirmake "/var/mail/${domain}/${user}"
    fi

I think we don't need the comment.
I probably should remove some outputs but it's another story.

Comment (spacecowboy) [inline start-mailserver.sh]: Sure. It was nice to have during development is all. Can remove it

**Comment by spacecowboy** [comment]
@tomav fixed your comment

**Comment by tomav** [comment]
Thank you!

**Pull request merged**